pubsub/api/Pubsub.Samples.Tests/CreatePushSubscriptionTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(PubsubFixture))]
+public class CreatePushSubscriptionTest
+{
+    private readonly PubsubFixture _pubsubFixture;
+    private readonly CreatePushSubscriptionSample _createPushSubscriptionSample;
+    private ListSubscriptionsSample _listSubscriptionsSample;
+
+    public CreatePushSubscriptionTest(PubsubFixture pubsubFixture)
+    {
+        _pubsubFixture = pubsubFixture;
+        _createPushSubscriptionSample = new CreatePushSubscriptionSample();
+        _listSubscriptionsSample = new ListSubscriptionsSample();
+    }
+
+    [Fact]
+    public void TestCreatePushSubscription()
+    {
+        string randomName = _pubsubFixture.RandomName();
+        string topicId = $"testTopicForCreatePushSubscription{randomName}";
+        string subscriptionId = $"testSubscriptionForCreatePushSubscription{randomName}";
+        string pushEndpoint = "https://my-test-project.appspot.com/push";
```

Comment:
it need to be there.

Reply:
Subscription creation will succeed with a non-existent endpoint. 
But messages being pushed to this endpoint will not be delivered. 
Since this sample is to show how to create a push subscription, this should be okay. We can add a comment to remind the reader to use a valid push endpoint.